packages/manager/modules/pci/src/projects/project/storages/volume-backup/create/create.template.html

```diff
@@ -0,0 +1,157 @@
+<!--Guides-->
+<oui-header
+    class="mb-3"
+    data-id="create-volume-backup-header"
+    data-heading="{{:: 'pci_projects_project_storages_volume_backup_create_header' | translate }}"
+>
+    <oui-guide-menu
+        data-text="{{:: 'pci_projects_project_storages_volume_backup_create_guides_header' | translate }}"
+    >
+        <oui-guide-menu-item
+            data-ng-repeat="guide in $ctrl.guides track by $index"
+            data-href="{{:: guide.link }}"
+            data-on-click="$ctrl.trackClick(guide.id)"
+            data-external
+        >
+            <span data-ng-bind=":: guide.title"></span>
+        </oui-guide-menu-item>
+    </oui-guide-menu>
+</oui-header>
+<p
+    class="py-2"
+    data-translate="pci_projects_project_storages_volume_backup_create_description"
+></p>
+
+<!--Step 1 :: Select a volume to backup-->
+<div class="py-2">
+    <!--Title-->
+    <p
+        class="oui-heading_3 mb-2"
+        data-translate="pci_projects_project_storages_volume_backup_create_step_1_title"
+    ></p>
+
+    <!--Select a volume-->
+    <oui-field
+        data-label="{{::'pci_projects_project_storages_volume_backup_create_step_1_select_label' | translate}}"
+        data-size="l"
+    >
+        <oui-select
+            data-disabled="$ctrl.isCreating || $ctrl.volumes.length === 0"
+            data-items="$ctrl.volumes"
+            data-match="name"
+            data-model="$ctrl.volumeBackupModel.selected.volume"
+            data-name="block-storage-volume"
+            data-placeholder="{{ 'pci_projects_project_storages_volume_backup_create_step_1_select_placeholder' | translate }}"
+        >
+        </oui-select>
+    </oui-field>
+</div>
+
+<!--Step 2 :: Select type of backup-->
+<div class="py-2">
+    <!--Title-->
+    <p
+        class="oui-heading_3 mb-2"
+        data-translate="pci_projects_project_storages_volume_backup_create_step_2_title"
+    ></p>
+
+    <!--Description-->
+    <p>
+        <span
+            class="d-block"
+            data-translate="pci_projects_project_storages_volume_backup_create_step_2_description_part_1"
+        ></span>
+        <span
+            class="d-block"
+            data-translate="pci_projects_project_storages_volume_backup_create_step_2_description_part_2"
+        ></span>
```

Comment:
**nitpick:** `<p class="mb-0"` instead of `<span class="d-block"`
```suggestion
        <p
            class="mb-0"
            data-translate="pci_projects_project_storages_volume_backup_create_step_2_description_part_1"
        ></p>
        <p
            class="mb-0"
            data-translate="pci_projects_project_storages_volume_backup_create_step_2_description_part_2"
        ></p>
```

Reply:
Any issue to use the `d-block` ?
Regarding BS documenation it's not deprecated.

Reply:
i have this structure :
```
<p>
   <span>....</span>
   <span>....</span>
   <a> .... </a>
```

Reply:
`span` is an inline element. Why to make it a block element with explicit `.d-block`.
maybe you can use 
```
<div>
    <p class="mb-0"></p>
    <p class="mb-0"></p>
    <a></a>
</div>
```